message-dispatcher/message-dispatcher.bal

```diff
@@ -0,0 +1,21 @@
+import ballerina/http;
+
+final http:Client ocrClient = check new ("http://api.ocr.com.balmock.io");
+
+type OcrResponse record {|
+    string[] lines;
+    int noOfLines;
+    string pdfUrl;
+|};
+
+service / on new http:Listener(8080) {
+    string[] ocrProcessors = ["processor1", "processor2", "processor3"];
```

Comment:
I think this array has to be final and readonly. Defining this at the module level also would be better.